dss_metrics/metrics.py

```diff
@@ -167,8 +172,34 @@ def get_metrics(self):
     # merge configs
     configs.update(cli_args)
 
+    # create logger
+    logfile_path = configs["logging_file"]
```

Comment:
Looks like you are creating two log files, one log file for main() and other log file for the collector(). Is that true?